src/misc/messages.cpp

```diff
@@ -193,19 +193,23 @@ static bool load_message_file(const std_fs::path &filename)
 	return true;
 }
 
-const char *MSG_Get(char const *requested_name)
+const char* MSG_Get(const char* requested_name)
 {
 	const auto it = messages.find(requested_name);
-	if (it != messages.end())
+	if (it != messages.end()) {
 		return it->second.GetRendered();
+	}
+	LOG_WARNING("LANG: Message '%s' not found", requested_name);
 	return msg_not_found;
 }
 
-const char* MSG_GetRaw(char const *requested_name)
+const char* MSG_GetRaw(const char* requested_name)
 {
 	const auto it = messages.find(requested_name);
-	if (it != messages.end())
+	if (it != messages.end()) {
 		return it->second.GetRaw();
+	}
+	LOG_WARNING("LANG: Message '%s' not found", requested_name);
```

Comment:
Is there value in differentiating rendered vs. raw in the log message?

Reply:
Yes. Rendered strings have Staging tags converted to ANSI sequences.

Reply:
Yes. Rendered have Staging markup tags converted to ANSI sequences and are converted to current DOS code page - ready for display. Raw ones are… raw - suitable for writings as comments into dosbox-staging.conf.